modules/Emails/EmailUI.php

```diff
@@ -422,6 +422,24 @@ public function generateComposePackageForQuickCreateFromComposeUrl($emailLinkUrl
         return $this->generateComposePackageForQuickCreate($a_composeData, $emailLinkUrl, $lazyLoad);
     }
 
+    private function createEmailLink($module_name, $record_id, $name, $addr, $text) {
```

Comment:
Can you please add phpdoc above this PR to see the acceptable type of arguments and return value.